carshare/settings.py

```diff
@@ -265,3 +266,9 @@ def env_get_list(variable, default):
         },
     },
 }
+
+# contact details
+CONTACT_PHONE = '01720 575 301'
+CONTACT_CARSHARE_EMAIL = 'carshare@ioscv.co.uk'
+CONTACT_EMAIL = 'info@ioscv.co.uk'
+CONTACT_INTERNATIONAL_PHONE = '+441720575301'
```

Comment:
Let's get rid of this one and just use the UK format phone number everywhere.

Reply:
There is one place where a tel href is used, I believe this needs the number in this format to work.

Reply:
Ahhh that makes sense.